<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatroom_main</class>
 <widget class="QWidget" name="chatroom_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white; color: black;border : 1px solid skyblue;</string>
  </property>
  <widget class="QWidget" name="top_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1080</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1080</width>
     <height>50</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1000</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="close_btn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="chat_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1080</width>
     <height>750</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1080</width>
     <height>750</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1080</width>
     <height>750</height>
    </size>
   </property>
   <widget class="QTextEdit" name="textEdit_send">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>603</y>
      <width>1074</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>999</x>
      <y>582</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QWidget" name="line" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>603</y>
      <width>1078</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 20px solid skyblue;
background-color: skyblue;</string>
    </property>
   </widget>
   <widget class="QWidget" name="line_2" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>580</y>
      <width>1078</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 20px solid skyblue;
background-color: skyblue;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1081</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
